src/ocl/convolutionocl.cpp

```diff
@@ -4502,13 +3721,7 @@ void ConvolutionDescriptor::ConvolutionWrwImmediate(Handle& handle,
         auto v_chk_kernels = std::vector<KernelInvoke>{chk_kernels.begin(), chk_kernels.end()};
         if(!v_chk_kernels.empty())
         {
-            MIOPEN_LOG_I2(
```

Comment:
Almost all code after `BackwardWeightsGemm()` is useless. If we do not have Invoker and this is not GEMM, then throw.